File: Common.Models/CardModel.cs
using System;
using System.Collections.Generic;
using System.Text;

 namespace Common.Models
{
    public class CardModel
    {
        public int Id { get; set; }
        
        public string Title { get; set; }

        public string Description { get; set; }

        public string SubDescription { get; set; }

        public byte[] Picture { get; set; }

        public int Price { get; set; }

        public IEnumerable<PlatformModel> Platforms { get; set; }

        public IEnumerable<GenreModel> Genres { get; set; }
    }
}

File: FE/src/models/CardModel.ts
import IGenreModel from "./GenreModel";
import IPlatformModel from "./PlatformModel";

export default interface ICardModel {
  id: number;
  title: string;
  description: string | null;
  picture: string;
  price: number;
  subDescription: string | null;
  platforms: IPlatformModel[] | null;
  genres: IGenreModel[] | null;
}
